API: Pass Entity Metadata directly instead of Repository for gqlArgsToMikroOrmQuery (#3727)
vivid-planet/comet: .changeset/silver-cloths-warn.md

---
"@comet/cms-api": major
---

> Pass Entity Metadata directly instead of Repository for MikroOrm Query

Reviewer: ```suggestion
Directly pass the entity metadata instead of the repository in `gqlArgsToMikroOrmQuery`
```
